cmd/common/helpers.go

```diff
@@ -0,0 +1,29 @@
+package common
+
+import (
+	"context"
+
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+// UpdateSecret updates a specified k8s secret with the provided data
+func UpdateSecret(kubeClient client.Client, secretName string, secretNamespace string, secretBody map[string][]byte) error {
```

Comment:
For my own better understanding: does this need to be exported? Or is it done for some reason I'm not seeing?  Or does it even matter?

I guess because it's in /common/, even if it's not yet used outside that package?

Reply:
I does need to be exported. It's being used outside of common: https://github.com/openshift/osd-utils-cli/pull/66/files#diff-2e4c77053adcf10316f9947fd2d309faR197